src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -126,6 +126,8 @@ export class CalciteRadioButton {
   //
   //--------------------------------------------------------------------------
 
+  private documentRoot: any;
```

Comment:
Could you rename this to avoid using `document`?

Reply:
How does `rootNode` sound?